cli.js

```diff
@@ -1,60 +1,64 @@
 #!/usr/bin/env node
 
-var pipboylib = require('pipboylib')
-var relay = require('pipboylib/lib/relay')
+var pipboylib = require('pipboylib');
+var relay = require('pipboylib/lib/relay');
```

Comment:
Amusingly, a friend of mine lambasted me for going the no-semicolon route while another praised it.. I actually prefer them in, so I welcome this. There's even a [linter for this style](https://github.com/rgbkrk/obama) with a presidential seal of approval.